### PR TITLE
Moves how to contribute to its own file

### PR DESCRIPTION
**Pull request by albertpastrana**
Signed-off-by: Albert Pastrana <albert.pastrana@gmail.com>

### CODE REVIEW

File: CONTRIBUTING [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Contributing to Anon
```
Comment (kittsville) [inline CONTRIBUTING]: Might be best to add `.md` so GitHub formats correctly.

**Comment by kittsville** [comment]
Wait, no. There's actual changes needed here.

File: README.md [hunk 2]
```diff
@@ -100,20 +100,6 @@ In order to be useful, Anon needs to be told what you want to do to each column
 }
 ```
 
-## How to contribute
```
Comment (kittsville) [inline README.md]: A lot of repos leave a contributing summary in the main readme e.g. [Scala](https://github.com/scala/scala), [Akka](https://github.com/scala/scala#how-to-contribute) and [Chronos](https://github.com/mesos/chronos#contributing). Could have a 2-3 line summary or just a pointer to the contributing file itself, YMMV.

File: CONTRIBUTING (hunk 1 above)
Comment (albertpastrana) [reply CONTRIBUTING]: sure, I forgot, thanks :)

File: README.md (hunk 2 above)
Comment (albertpastrana) [reply README.md]: good point

**Pull request merged**